db/disk.go

```diff
@@ -44,3 +49,72 @@ func initDisks(tx *sql.Tx) error {
 
 	return err
 }
+
+func (d DB) CreateDisk(vendor string,model string,
+	machine *Machine, capacity int, ssd bool) error {
+	_, err := d.sql.Exec(`
+		INSERT INTO Disks(vendor, model, machine, capacity, ssd)
+		VALUES(
+			?,
+			?,
+			?,
+			?,
+			?
+			)`,
+		 vendor, model, machine.Id, capacity, ssd)
+		 //fmt.Printf("", machine)
+
+	return err
+}
+
+
+func (d DB) GetDisks() ([]Disk, error){
+	rows, err := d.sql.Query(`
+		SELECT id, vendor, model, machine, capacity, ssd
+		FROM Disks 
+		ORDER BY capacity DESC
+
+	`)
+	if err != nil {
+		return nil, err
+	}
+
+	defer rows.Close()
+
+	disks := []Disk{}
+	for rows.Next() {
+		k := Disk{db: &d}
+		var machine int
+			err = rows.Scan(&k.id, &k.Vendor,
+			&k.Model, &machine, &k.CapacityGB, &k.SSD)
+		if err != nil {
+			return nil, err
+		}
+			if machine != 0{
+		k.Machine, err = k.db.GetMachine("id", machine)
+				fmt.Printf("", machine)
+			}
+
+		disks = append(disks, k)
+	}
+
+	return disks, rows.Err()
+
+}
+func (k Disk) machine() (*Machine, error) {
+	return k.db.GetMachine("id", k.Machine)
+}
+
+func (k Disk) String() string {
+
+	var machineName string
+	Machine, err := k.machine()
+	if err != nil {
+		machineName = fmt.Sprintf("<error: %s>", err)
+	} else {
+		machineName = Machine.Name
+	}
+
+	return fmt.Sprintf("%-s %-s %-s %d GB RAM %t",
```

Comment:
`%t` isn't very helpful on its own: you'll sometimes see `true` printed out and sometimes `false`, but with no indication of what is true or false. It would be better to print out something like `SSD` or `HDD` (or even `SSD` or a blank space).